website/static/js/filepage/revisions.js

```diff
@@ -120,6 +126,20 @@ var FileRevisionsTable = {
                         }
                     }, m('i.fa.fa-download'))
                 ),
+                model.hasHashes ? m('td',
+                    m('div.input-group[style="width: 180px"]',
+                        [
+                            m('span.input-group-btn', m('button#copyBtnMd5'+ revision.displayVersion +'.btn.btn-default.btn-sm[type="button"][data-clipboard-text="'+revision.extra.hashes.md5 + '"]', {config: new makeClient($('#copyBtnMd5'+ revision.displayVersion))}, m('.fa.fa-copy'))),
```

Comment:
The `config` param should be a function; you are passing it a ZeroClipboard client object. Notice that `makeClient` is not a constructor function. It is a factory, so the `new` keyword is incorrect usage.

Also, `makeClient` takes a native DOM element object (not a jQuery object). Conveniently, the `config` function receives the DOM element, so you can pass that to `makeClient`. This also means you can remove the unnecessary CSS IDs.